c_glib/arrow-glib/table.cpp

```diff
@@ -18,7 +18,7 @@
  */
 
 #ifdef HAVE_CONFIG_H
-#  include <config.h>
+#include <config.h>
```

Comment:
Please revert the formatting changes. If there are any changes to table.cpp which aren't formatting I can't see them easily.

Reformatting is valid but if it needs to be done at scale then I'd prefer it be done in a dedicated PR